src/components/InteractionLayer/index.js

```diff
@@ -199,15 +167,70 @@ class InteractionLayer extends React.Component {
     }
   };
 
+  processMouseMove = (
```

Comment:
Why does this need to have `props` as input? It has access to `this.props` itself.

Reply:
Good point, will fix that